nixos/modules/services/web-apps/immich.nix

```diff
@@ -352,8 +377,8 @@ in
         IMMICH_MEDIA_LOCATION = cfg.mediaLocation;
         IMMICH_MACHINE_LEARNING_URL = "http://localhost:3003";
       }
-      // lib.optionalAttrs (cfg.settings != null) {
-        IMMICH_CONFIG_FILE = "${format.generate "immich.json" cfg.settings}";
+      // lib.optionalAttrs (cfg.settings != null || cfg.settingsFile != null) {
```

Comment:
uhm, afaict settingsFile isn't an option in this module. Should this have been secretsFile ?

Reply:
I'm very sorry about that. Fixed in https://github.com/NixOS/nixpkgs/pull/449271.